velero-plugin-for-aws/volume_snapshotter.go

```diff
@@ -17,15 +17,17 @@ limitations under the License.
 package main
 
 import (
+	"context"
 	"fmt"
 	"os"
 	"regexp"
 	"strings"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/awserr"
-	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/aws/smithy-go"
+
```

Comment:
Looking at a diff from this PR to upstream release-1.10, these imports are added in a different place, so the resulting PR will be moving imports. Best to put them in the same place here as they are upstream so it doesn't add to future merge conflict points.